onnxruntime/contrib_ops/cuda/diffusion/group_norm.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include "core/common/common.h"
+#include "core/providers/cuda/cuda_kernel.h"
+
+namespace onnxruntime {
+namespace contrib {
+namespace cuda {
+
+using namespace onnxruntime::cuda;
+
+template <typename T>
```

Comment:
>emplate <typename T> [](http://example.com/codeflow?start=1&length=20)

Can we make only ComputeInternal() templated? The constructor does not need a template, does it?

Reply:
updated class to no template.